test/unit/client/system/browserify.test.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+var path    = require('path'),
+  vm      = require('vm'),
+  fs      = require('fs'),
+  should  = require('should'),
+  ss      = require( '../../../../lib/socketstream'),
+  options = ss.client.options;
+
+describe('browserify', function() {
+
+  var browserify = fs.readFileSync( path.join(__dirname,'../../../..', 'lib/client/system/libs/browserify.js')),
+      browser;
+
+  describe('require()', function () {
+
+    beforeEach(function() {
+      browser = (function(){
+        var window = {document:{}};
+        window.window = window;
+        vm.runInNewContext(browserify, window);
+
+        return window;
+      })();
+    });
+
+    it('should be sane',function() {
+      should(browser.require).be.type('function');
+    });
+
+    it('should resolve require with absolute path', function() {
+      browser.require.define('/a/b/c/d',
+        function(require, module, exports, __dirname, __filename) {
```

Comment:
'__filename' is defined but never used.<br>'__dirname' is defined but never used.